src/classes/CourseConSettingHealthCheckVMapper.cls

```diff
@@ -62,35 +62,59 @@ public virtual with sharing class CourseConSettingHealthCheckVMapper {
     ********************************************************************************************/ 
     public virtual HealthCheckGroupVModel getHealthCheckGroup() {
 
-        HealthCheckGroupVModel hlthChkGroupVMod = new HealthCheckGroupVModel(
-            Label.stgHCCourseConnectionsTitle,
-            Label.stgHealthCheckStatusPassed,
-            0,
-            0,
-            new List<String>(),
-            new List<HealthCheckItemVModel>()
-        );
-
         CourseConnectionSettingsModel courseConnSettingsModel = this.locateCourseConnectionSettingsService().getCourseConnectionSettingsModel();
 
         String objectName = UTIL_Namespace.StrTokenNSPrefix('Course_Enrollment__c');
         SObjectType courseConnSObjectType = UTIL_Describe.getObjectDescribe(objectName).getSObjectType();
         Map<Id, RecordTypeModel> courseConnRTModelsById = this.locateRecordTypeService().getModelByIdMapForSObjectType(courseConnSObjectType);
 
-        HealthCheckGroupVModel courseConnEnableCheckResult = this.getCourseConnectionsNotEnabledHealthCheckItems(courseConnSettingsModel);
+        List<HealthCheckGroupVModel> courseConSettingtHlthCheckGroupVModelsList = new List<HealthCheckGroupVModel>();
+        courseConSettingtHlthCheckGroupVModelsList.add(this.getCourseConnectionsNotEnabledHealthCheckItems(courseConnSettingsModel));
```

Comment:
You'll need to do more with this specific item as it should exclude the following three if it is in a warning status. This is missing the requirement of skipping the next three checks if it comes back as a warning.